HomeUI/src/views/explorer/Transaction.vue

```diff
@@ -178,43 +178,43 @@
         classes="skinny-list-entry"
       />
       <list-entry
-        v-if="transaction.type === 'Starting a zelnode'"
+        v-if="transaction.type === 'Starting a FluxNode'"
         title="Collateral Public Key"
         :data="transaction.collateral_pubkey"
         classes="skinny-list-entry"
       />
       <list-entry
-        v-if="transaction.type === 'Starting a zelnode'"
+        v-if="transaction.type === 'Starting a FluxNode'"
         title="Flux Public Key"
-        :data="transaction.zelnode_pubkey"
+        :data="transaction.zelnode_pubkey || transaction.fluxnode_pubkey || transaction.node_pubkey"
         classes="skinny-list-entry"
       />
       <list-entry
-        v-if="transaction.type === 'Confirming a zelnode'"
+        v-if="transaction.type === 'Confirming a FluxNode'"
         title="Flux Network"
         :data="transaction.ip"
         classes="skinny-list-entry"
       />
       <list-entry
-        v-if="transaction.type === 'Confirming a zelnode'"
+        v-if="transaction.type === 'Confirming a FluxNode'"
         title="Update Type"
         :number="transaction.update_type"
         classes="skinny-list-entry"
       />
       <list-entry
-        v-if="transaction.type === 'Confirming a zelnode'"
+        v-if="transaction.type === 'Confirming a FluxNode'"
```

Comment:
Not sure about this one.... :D

Reply:
Checking :+1:

Reply:
Capitalization fixed. Actually current home production ui has it wrongly and cant show the data. As this type is obtained from daemon where it its now referenced already as 'fluxnode'